android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -2266,10 +2266,6 @@ public void setFullscreen(boolean fullscreen) {
                 fullScreenPlayerView.show();
             }
             UiThreadUtil.runOnUiThread(() -> {
-                originalFitsSystemWindows = window.getDecorView().getFitsSystemWindows();
```

Comment:
but why did you remove this code ?

Reply:
I have decided to remove the system window handling code `(fitsSystemWindows and WindowCompat.setDecorFitsSystemWindows)` because fullscreen mode is entirely managed within a Dialog. Since the dialog controls how the UI behaves in fullscreen mode, we no longer need to manually adjust the system windows at the activity level when entering or exiting fullscreen. The dialog itself will handle all necessary UI changes, including the display and behavior of system bars. Therefore, maintaining redundant fullscreen handling in the activity is unnecessary.

Reply:
Exactly 
I will remove it.

Reply:
@freeboub 
I removed it.